lms/djangoapps/course_api/blocks/tests/test_views.py

```diff
@@ -28,8 +29,23 @@ def setUpClass(cls):
         super(TestBlocksView, cls).setUpClass()
 
         # create a toy course
-        cls.course_key = ToyCourseFactory.create().id
+        cls.course = ToyCourseFactory.create()
+        cls.course_key = cls.course.id
         cls.course_usage_key = cls.store.make_course_usage_key(cls.course_key)
+        cls.chapter = ItemFactory.create(
```

Comment:
Have you thought about just updating one of the existing items in ToyCourseFactory to include a `due` field?